README.md

```diff
@@ -1 +1,238 @@
 # Sync and Async
+
+## Overview
+
+We've spent a lot of time in the beginning of Unit 1 talking about non-blocking input/output. While this pattern is great for performance, sometimes you will need to write synchronous code in Node.
+
+Yes, synchronous code might be useful. Imagine a situation where you read configurations from a file to boot up your system. Without the configurations, the system won't work. In this case, you don't want asynchronous code. The synchronous pattern would be a better idea, because the system won't start without the configurations.
+
+On the other hand, one of the major challenges in learning Node.js is its asynchronous code patterns. Human brain just didn't evolve to think in parallel processes. This hurdle is exacerbated when people learning Node are experienced developers in other more traditional and synchronous languages (PHP, Java, Ruby, Python, C).
```

Comment:
This paragraph doesn't really flow from the previous one. A better contrast might be performing lots of expensive operations whose results don't depend on each other.